opt.gunk

```diff
@@ -1,13 +1,18 @@
 // Package opt provides the standard gunk protobuf options.
-// 
+//
 // Generated by the gunkconv tool.
 package opt
 
 // Deprecated is the deprecated option.
 type Deprecated bool
 
-// JavaPackage is the java_package option.
-type JavaPackage string
+// OptimizeFor is the optimize_for option.
+type OptimizeFor int
 
-// JavaMultipleFiles is the java_multiple_files option.
-type JavaMultipleFiles bool
+// Values for OptimizeFor enum.
+const (
+    NotSet OptimizeFor = iota
```

Comment:
This is required because otherwise we get the following error with protoc: `github.com/gunk/opt/all.proto: The first enum value must be zero in proto3.` However, this `NotSet` field isn't present in the description.proto definition of `FileOption`. I am unsure if we should leave this as it is, or add a "work-around" in gunk to always ignore enums from `github.com/gunk/opt`. This is happening because gunk will add all these types into the final proto definition, which I am not sure we actually need?

Reply:
These enums shouldn't be in the final unified proto file, no. Leave a TODO here and open an issue on gunk, then we can fix it there.

Reply:
It is not just the enums, but anything defined in these packages, however we currently drop enums that don't have any enum values associated with them. I'll raise an issue and link it in a comment here.

Reply:
Done.

Reply:
The fix for this was merged into master https://github.com/gunk/gunk/commit/f49402038d65722049122f7342e1d8312c2e614f. This is no longer causing errors.